st2common/st2common/middleware/error_handling.py

```diff
@@ -62,7 +62,7 @@ def __call__(self, environ, start_response):
             if isinstance(e, exc.HTTPException):
                 status_code = status
                 message = six.text_type(e)
-            elif isinstance(e, db_exceptions.StackStormDBObjectNotFoundError):
+            elif isinstance(e, db_exceptions.StackStormDBObjectNotFoundError) or isinstance(e, db_exceptions.StackStormDBKeyNotFoundError):
```

Comment:
As we extend DBObjetNotFoundError we  no-longer need this or on isinstance of StackStormDBKeyNotFoundError